src/Core/CoreKernel.php

```diff
@@ -87,31 +87,40 @@ protected function bootDatabaseGlobals()
         global $databaseConfig;
         global $database;
 
-        // Case 1: $databaseConfig global exists. Merge $database in as needed
-        if (!empty($databaseConfig)) {
-            if (!empty($database)) {
-                $databaseConfig['database'] =  $this->getDatabasePrefix() . $database . $this->getDatabaseSuffix();
+        for ($i = 0; $i <= 99; $i++) {
```

Comment:
Might pay to make a const on `DB` - something like `public const MAX_REPLICAS = 99;` since we have this `99` number in a few places.

Reply:
Updated